grakn-graph/src/test/java/ai/grakn/graph/internal/TxCacheTest.java

```diff
@@ -134,6 +135,21 @@ public void whenDeletingAnInstanceWithNoRelations_EnsureLogIsEmpty(){
     }
 
     @Test
+    public void whenNoOp_EnsureLogNotEmpty() {
```

Comment:
Personally I think the title of this test is misleading. The Log is empty, it contains no useful information.